src/SprykerEco/Zed/Ratepay/Business/Order/PartialOrderCalculatorInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: sikachev
+ * Date: 6/23/17
+ * Time: 11:22 AM
+ */
+
+namespace SprykerEco\Zed\Ratepay\Business\Order;
+
+interface PartialOrderCalculatorInterface
+{
+
+    /**
+     * @param int $idSalesOrderItem
+     *
+     * @return \Generated\Shared\Transfer\ItemTransfer
+     */
+    public function getOrderItemTotalsByIdSalesOrderItem($idSalesOrderItem);
+
+}
```

Comment:
I guess, I removed it by hands. Other files created by me seem to have new lines.